lib/src/ruby/fact.cc

```diff
@@ -120,6 +120,21 @@ namespace facter { namespace ruby {
                 // Set the value to what was resolved
                 _value = value;
                 _weight = weight;
+
+                if (! ruby.is_nil(_value)) {
+                  return 0;
+                }
+
+                // None of our resolvers were able to resolve the value.
+                // Thus, we attempt to use the "built-in" fact's value.
+                auto builtin_value = facts[ruby.to_string(_name)];
+                if (builtin_value) {
+                  // Already in collection, do not add
+                  add = false;
+                  _value = facter->to_ruby(builtin_value);
```

Comment:
I don't really like wrapping the builtin fact as a ruby one - I'd much rather handle the fallback somewhere else, if we can

Reply:
Ah, nevermind, I see we already do this wrapping for other cases when we don't add. Weird.